packages/venia-ui/lib/components/CartPage/PriceAdjustments/ShippingMethods/__tests__/shippingMethods.spec.js

```diff
@@ -23,7 +23,7 @@ jest.mock('../shippingForm', () => 'ShippingForm');
 jest.mock('../shippingRadios', () => 'ShippingRadios');
 
 test('renders description and confirm link w/o shipping address set', () => {
-    useLazyQuery.mockReturnValue([
+    useQuery.mockReturnValue([
```

Comment:
I'm surprised this is working - `useQuery` just returns an object, not an array like `useLazyQuery` does.

Reply:
Very odd. I haven't full covered this with tests yet, so I'll be sure to investigate this and update the mock to accurately reflect the implementation of useQuery.